pages/pipelines/block_step.md.erb

```diff
@@ -52,6 +56,20 @@ _Optional attributes:_
       <em>Example:</em> <code>build.message != "skip me"</code>
     </td>
   </tr>
+  <tr>
+    <td><code>depends_on</code></td>
+    <td>
+      A list of step keys that this step depends on. This step will only run after the defined step has completed. See <a href="/docs/pipelines/dependencies">managing step dependencies</a> for more information.<br>
+      <em>Example:</em> <code>"test-suite"</code>
+    </td>
+   </tr>
+   <tr>
+    <td><code>allow_dependency_failure</code></td>
+    <td>
+      Whether to continue to run this step if any of the steps defined in the <code>depends_on</code> attribute fail.<br>
```

Comment:
I think leave it as is for now, we can always add it in later if people are asking questions about it. I think it would get too busy if we tried to smush the extra info in there.